index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <style>
+      @media only screen and (max-width: 600px) {
+        body {
+          background-color: lightblue;
+        }
+      }
+    </style>
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <section id="welcome-section">
+      <h1>This is a portfolio</h1>
+    </section>
+    <section id="projects">
+      <div class="project-tile">project title</div>
+      <a href="https://www.weatherapi.com/">My weather API</a>
+    </section>
+    <navbar id="navbar">
+      <a href="#projects">Projects</a>
+      <br />
+    </navbar>
+    <a
+      href="https://github.com/codedit334/microverse_trials_emmanuel_oussama"
+      id="profile-link"
+      target="_blank"
+      >GitHub</a
```

Comment:
- [ ] It is a good practice to use class instead of ID in multiple elements

Reply:
- [ ] Nice job pointing out this issue! It provides a clear recommendation for using `class` instead of `ID` in multiple elements, which can improve the readability and maintainability of the code. However, it would be more effective if you could provide an **explanation or example** of **WHY** using class is better in this situation, as **this can help others understand the reasoning behind the recommendation and encourage them to adopt the best practice**. 🎯

Reply:
Done